.gitpod.Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM gitpod/workspace-full
```

Comment:
If we are using the default image, why do we need to create this docker container?